src/Data/ECamera.cs

```diff
@@ -0,0 +1,16 @@
+namespace Data
+{
+    using System.ComponentModel.DataAnnotations;
+
+    public class ECamera
+    {
+        [Required]
+        public string Id { get; set; }
+
+        [Required]
+        public int Latitude { get; set; }
+
+        [Required]
+        public int Longitude { get; set; }
+    }
+}
```

Comment:
Should we move the new entity under the `Data/Camera/Entities` folder? Should we do the same with EDetection?

Reply:
done